src/stores/opetussuunnitelma.ts

```diff
@@ -200,6 +203,7 @@ export class OpetussuunnitelmaStore {
   public async updateValidation() {
     this.lops2019Validation = null;
     this.validointi = null;
+    this.isValidating = true;
```

Comment:
Eikös tämä isValidating ole siinä mielessä turha, että tiedetään jo tuosta this.validointi:sta että onko haku kesken silloin jos this.validointi = null ?